config/routes.rb

```diff
@@ -9,7 +9,8 @@
       collection do
         # Create should be a POST, but this is what the Java WFS app did.
         put ':workflow', to: 'workflows#create'
-        put ':workflow/:process', to: 'workflows#update'
+        put ':workflow/:process', to: 'steps#update'
+        post ':workflow/:process/next', to: 'steps#next'
```

Comment:
PR is looking good so far. I'm curious about mixing `post` and `put` here. Does `steps#next` 
function more like a create operation? The methods look similar to me.